### ISSUE TITLE
Enable onBlur validation

### ISSUE THREAD

**Issue by djanosik** [riganti/dotvvm#277]
Validation of user input happens when a postback is made (form is sent, etc.). But it should happen when the input value is changed (in onBlur event). It will improve user experience.

- enable onBlur validation
- add `Validator.ValidCssClass` property
- implement more validators

**Comment by tomasherceg** [comment]
What about adding `Validation.Event` attached property with the following options:

* `Command` (default behavior)
* `Blur` (turns on on blur validation)

There might be more in the future.

**Comment by tomasherceg** [comment]
I know, but this feature is quite important and if we do it together with other validation enhancements (better extensibility etc.), it should be OK.

**Comment by martindybal** [comment]
I was wondering, Is there any progress on the issue?

**Comment by exyi** [comment]
@martindybal No, I'm not aware of any progress, we still have no idea how should this work and no one really wants to redesign the client-side validation.
